app/views/comments/create.js.erb

```diff
@@ -0,0 +1,5 @@
+var comments_list = $('#comments-list');
+comments_list.append('<%= j (render partial: @comment) %>');
+$('#comments-count').html('<%= j (pluralize(@post.comments.count, "comment")) %>');
+<% @comment = @new_comment %>
```

Comment:
Remove this line and pass it to form like `render("comments/form"), comment: @new_comment`